Bundle/WidgetMapBundle/Manager/WidgetMapManager.php

```diff
@@ -162,7 +160,13 @@ public function delete(View $view, Widget $widget)
             $view->addWidgetMap($replaceWidgetMap);
         }
 
+        //Move children for current WidgetMap View
         $this->moveChildren($view, $beforeChild, $afterChild, $originalParent, $originalPosition);
+
+        //Move children WidgetMap for children from other View
+        foreach ($widgetMap->getChildren() as $child) {
+            $this->moveWidgetMap($child->getView(), $child, $originalParent, $originalPosition);
```

Comment:
When you move a WidgetMap, the logic is already good to reattach children WidgetMap from inherited Views. We only need this additional logic when deleting a Widget from a Template.
You already did a good job on Widget moves @paulandrieux 👍